packages/hardhat-ovm/test/hardhat-project/hardhat.config.js

```diff
@@ -0,0 +1,11 @@
+require('../../dist/index')
+
+module.exports = {
+  solidity: '0.7.6',
+  networks: {
+    optimism: {
+      url: 'http://locahost:8545',
+      ovm: true,
```

Comment:
Unrelated to this PR but... it seems a bit weird to me that a network configuration affects the compiler. Wouldn't it be better if the overridden compile task had a `--ovm` flag?

Reply:
One concern is that if people are already using this then changes would break their config...